Sources/TensorFlow/Operators/NN.swift

```diff
@@ -434,6 +529,29 @@ public func conv3D<Scalar: TensorFlowFloatingPoint>(
         padding: padding.raw)
 }
 
+/// Computes a 2-D depthwise convolution with the specified input, filter, strides, and padding.
+///
+/// - Parameters:
+///   - x: The input.
+///   - kernel: The depthwise convolution filter.
+///   - strides: The strides of the sliding filter for each dimension of the input.
+///   - padding: The padding for the operation.
+/// - Precondition: `self` must have rank 4.
+/// - Precondition: `kernel` must have rank 4.
+@differentiable(wrt: (x, kernel), vjp: _vjpDepthwiseConv2D)
+public func depthwiseConv2D<Scalar: TensorFlowFloatingPoint>(
+    _ x: Tensor<Scalar>,
```

Comment:
Could you please apply what #246 did, which is to change all occurrences of `x` to `input`?